packages/cypress/src/utils/find-target-options.ts

```diff
@@ -64,65 +65,79 @@ function findInTarget(tree: Tree, options: FindTargetOptions) {
   if (!foundConfig) {
     return projectConfig?.targets?.[target];
   }
-  projectConfig.targets[target].options = {
+
+  const mergedOptions = {
     ...(projectConfig.targets[target]?.options || {}),
     ...projectConfig.targets[target]?.configurations[foundConfig],
   };
```

Comment:
Can this use readTargetOptions? e.g. `readTargetOptions({project, target configuration})`

Reply:
If not, you can remove the `|| {}`